recipes-qt4/qt4/qt4-4.8.7/0040-qt4-x11-free-messagemodel.cpp-comparison-patch.patch

```diff
@@ -0,0 +1,26 @@
+From fe547089114307b086167639066a1551cec39d11 Mon Sep 17 00:00:00 2001
+From: Edi Feschiyan <edi.feschiyan@konsulko.com>
+Date: Fri, 20 Oct 2023 11:13:28 +0000
+Subject: [PATCH] qt4-x11-free: messagemodel.cpp comparison patch
+
+Switching comparison from a pointer with integer
+to pointer with (void *)pointer comparison
+
```

Comment:
In a patch like this, we need to set [Upstream-Status](https://docs.yoctoproject.org/dev/contributor-guide/recipe-style-guide.html?highlight=upstream+status#patch-upstream-status). In this case, I will do it for you.

Upstream-Status: Inappropriate [Inactive upstream]

Which says there is no point in submitting the patch to upstream (because it has been EOL since 2015).